.gitmodules

```diff
@@ -42,3 +42,6 @@
 [submodule "modules/lwip"]
 	path = modules/lwip
 	url = https://github.com/ArduPilot/lwip.git
+[submodule "modules/littlefs"]
+	path = modules/littlefs
+	url = https://github.com/ArduPilot/littlefs.git
```

Comment:
Would much prefer to just vendor the two relevant files. Git submodules are always a pain though admittedly one more doesn't significantly increase the level.

Reply:
Yeah, I hear you but that ship has sailed and the extra files in the repo are actually somewhat useful. The repo is not big.